misc/write_fake_manifests.py

```diff
@@ -76,7 +76,7 @@ def path(self):
 
     def src_obj_pairs(self, path, name):
         num_sources = paretoint(55, alpha=2) + 1
-        return [(os.path.join('..', '..', path, s + '.cc'),
+        return [(os.path.join('src', path, s + '.cc'),
```

Comment:
Hm, having the build directory as a child of the src directory and the cwd during compilations is fairly common (it's how cmake, gyp, gn all work), so if at all possible I think it'd be good to keep the "../.." prefix on source file references.